pulp_2to3_migration/app/models/repository.py

```diff
@@ -175,7 +176,7 @@ class Pulp2Distributor(BaseModel):
                                               null=True)
 
     class Meta:
-        unique_together = ('pulp2_repository', 'pulp2_id')
+        unique_together = ('pulp2_object_id',)
```

Comment:
Do you remember why we had  `unique_together = ('pulp2_repository', 'pulp2_id')` ?
Is `pulp2_repository` always native to the distributor? or is it the repository from the plan?
If it's always native, then I think we can change the uniqueness.

Reply:
i don't remember why we had this constraint, in my understanding( and memory) pulp2_repository was always native to the distributor